<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>808</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="mainSplitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="robotSelect">
          <item>
           <widget class="QRadioButton" name="robot_1_Radio">
            <property name="text">
             <string>Robot 1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="robot_2_Radio">
            <property name="text">
             <string>Robot 2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="robot_3_Radio">
            <property name="text">
             <string>Robot 3</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="segLabel_1">
          <property name="text">
           <string>Segment Length &amp; Tendon Change (mm)

First segment:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="segLen_1">
          <item>
           <widget class="QSlider" name="segLenSlider_1">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="segLenBox_1"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_1_1">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_1_2">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_1_3">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_1_4">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="segLabel_2">
          <property name="text">
           <string>Second segment:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="segLen_2">
          <item>
           <widget class="QSlider" name="segLenSlider_2">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="segLenBox_2"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_2_1">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_2_2">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_2_3">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_2_4">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="segLabel_3">
          <property name="text">
           <string>Third segment:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="segLen_3">
          <item>
           <widget class="QSlider" name="segLenSlider_3">
            <property name="maximum">
             <number>9999</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="segLenBox_3"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_3_1">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_3_2">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_3_3">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="tendon_3_4">
          <property name="minimum">
           <double>-99.989999999999995</double>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="posePlotCheckBox">
          <property name="text">
           <string>Plot Pose</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calculateButton">
          <property name="text">
           <string>Calculate</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
